Tests/airport.spec.js

```diff
@@ -15,23 +15,21 @@ console.log('1. addTailNumber adds Plane to the Airport');
 
 //Arrange
 airport = new Airport();
-//plane = new Plane(); Got carried away, left in to show progression (I think :D)
+plane = new Plane();
 expected = 1;
 
 //Act
-airport.addPlane();
+airport.addPlane(plane);
 actual = airport.slotsOccupied.length;
-console.log(airport)
 
 //Assert
-result = testEquals(expected, actual) ? 'Yes' : 'No';
+result = testEquals(expected, actual) ? 'Yes, the Plane has landed at the airport' : 'No, the Plane is still in the sky';
```

Comment:
I added the ternary operator (I think that's what its called) to help me clearer understand the output I was getting from the code. Wouldn't need to use this anymore as over the week I developed a clearer understanding of how the code and more specifically my code worked.